### ISSUE TITLE
ctrl/cmd+shift+h does not work for chat question caraousel

### ISSUE THREAD

**Issue by anthonykim1** [microsoft/vscode#295242]
Its getting overriden by `search: replace in files`

https://github.com/user-attachments/assets/939c6e2c-9ba4-4b98-9c39-da61f5913c2e

https://github.com/user-attachments/assets/808a7a7c-90fe-4d53-8bc0-2908bc482188

@accesswatch @rperez030

**Comment by accesswatch** [comment]
@meganrogge I am still seeing ctrl+shift+h taking affect in chat, is that expected? Do I need to reset settings or remove a file to get this configuration?

**Comment by accesswatch** [comment]
@anthonykim1 I am still not able to get this keystroke to work in the latest Insider builds. Any hint on this by chance?
